shared/cfg/guest-os/Windows/Win2008/i386.cfg

```diff
@@ -1,5 +1,4 @@
 - i386:
-    maxmem = 4G
```

Comment:
For "Windows Server 2008 Datacenter" is 64GB according to this doc above . I think it is no need to limit it.
We always install datacenter os not "Standard" or other systems.